web/ui/src/idai_field/projects.ts

```diff
@@ -1,15 +1,19 @@
 import { ResultDocument } from '../api/result';
 import { Document } from '../api/document';
+import { getTranslation } from '../shared/languages';
 
 
 const MAX_LABEL_LENGTH = 35;
 
 
 export const getProjectLabel = (projectDocument: ResultDocument|Document): string => {
 
-    return projectDocument.resource.shortName ??
-        (projectDocument.resource.shortDescription
-            && projectDocument.resource.shortDescription.length <= MAX_LABEL_LENGTH
-                ? projectDocument.resource.shortDescription
+    const shortDescription = getTranslation(projectDocument.resource.shortDescription);
```

Comment:
resource.shortDescription kann leer sein -> also soll getTranslation keinen Error schmeißen

Reply:
Ich lasse getTranslation jetzt immer '' zurückgeben (siehe unten in `languages.ts`)